bucketlist/app/forms.py

```diff
@@ -0,0 +1,80 @@
+from django import forms
+from .models import User, BucketList, BucketListItem
+from django.contrib.auth import(
+    login,
+    logout,
+)
+from django.contrib.auth.models import User
+
+
+class SignUpForm(forms.ModelForm):
+    """Registration form validation for signing up a user"""
+    def __init__(self, *args, **kwargs):
+        super(SignUpForm, self).__init__(*args, **kwargs)
+
+        for fieldname in ['username', 'email', 'password', 'password_two']:
+            self.fields[fieldname].help_text = None
+    
+    password_two = forms.CharField(widget=forms.PasswordInput())
```

Comment:
*Nitpick*: I would prefer `password_conf`. Makes a lot more sense for this variable.